src/main/java/net/pms/dlna/RootFolder.java

```diff
@@ -1477,8 +1477,8 @@ public void notify(String filename, String event, FileWatcher.Watch watch, boole
 						RealFile rf = new RealFile(file);
 						rf.setParent(rf);
 						rf.getParent().setDefaultRenderer(RendererConfiguration.getDefaultConf());
-
 						rf.resolveFormat();
+						rf.syncResolve();
```

Comment:
@SubJunk I thought this was a lot of extra effort just to trigger the OpenSubtitles lookup, so I wrote an alternative solution. But, when testing I found that ```syncResolve``` should be called in any case, because that't the call that actually parses the media. It's much better to parse the media *and* call OpenSubtitles than to only call OpenSubtitles. I therefore conclude that this is a good solution.

What I don't understand is how this worked properly before I moved "the trigger", as the trigger was then in ```LibMediaInfoParser.parse()``` which wouldn't be called from this method...?